redis/src/push_manager.rs

```diff
@@ -0,0 +1,61 @@
+use crate::{PushKind, RedisResult, Value};
+use arc_swap::ArcSwap;
+use std::sync::Arc;
+
+/// Holds information about received Push data
+#[derive(Debug, Clone)]
+pub struct PushInfo {
+    /// Push Kind
+    pub kind: PushKind,
+    /// Data from push message
+    pub data: Vec<Value>,
+
+    /// Connection address to distinguish connections
+    pub con_addr: Arc<String>,
+}
+
+/// Manages Push messages for both tokio and std channels
+#[derive(Clone, Default)]
+pub struct PushManager {
```

Comment:
I know the implementation's pretty simple and self evident here, but please add a couple of tests to this file - send message, don't send after sender was replaced, can send messages after receiver was dropped, etc.

Reply:
I have just written some tests, including a multitasking test for channels and messages.